api/src/pcapi/core/token/__init__.py

```diff
@@ -94,9 +106,44 @@ def create(cls, type_: TokenType, ttl: timedelta | None, user_id: int, data: dic
         token._log(cls._TokenAction.CREATE)
         return token
 
+
+@dataclasses.dataclass(frozen=True)
+class SixDigitsToken(AbstractToken):
     @classmethod
-    def token_exists(cls, type_: TokenType, user_id: int) -> bool:
-        return app.redis_client.exists(cls._get_redis_key(type_, user_id))  # type: ignore [attr-defined]
+    def _get_redis_extra_data_key(cls, type_: TokenType, user_id: int) -> str:
+        return f"pcapi:token:data:{type_.value}_{user_id}"
 
-    def _log(self, action: _TokenAction) -> None:
-        logger.info("[TOKEN](%s){%i, %s, %s}", action.value, self.user_id, self.type_.value, self.encoded_token)
+    @classmethod
+    def load_without_checking(
+        cls,
+        encoded_token: str,
+        type_: TokenType,
+        user_id: int,
+    ) -> "SixDigitsToken":
+        try:
+            data = json.loads(app.redis_client.get(cls._get_redis_extra_data_key(type_=type_, user_id=user_id)))  # type: ignore [attr-defined]
+        except:
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏except doit attraper une exception